.github/scripts/check_config_changes_ci.py

```
@@ -581,24 +655,24 @@ def main():
                 print(f"     {c}")
         else:
             print(f"  –  {r.label}: no changes")
- 
+
     note = build_pr_note(results)
     if not note:
         print("\nℹ️  No notable changes found across all checkers.")
```

Ruff (0.16.1)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)